### ISSUE TITLE
Decide on channels to release our accessibility tips every week

### ISSUE THREAD

**Issue by 1Copenut** [adhocteam/a11y-snippets#16]
## Description
We should rally on the communication channels we want to release our accessibility tips every week. I'm thinking:
* Slack (respective right channels for programs)
* Ad Hoc Twitter (need guidance from @dannychapman)

Any others?

## Related Issue
* https://github.com/adhocteam/a11y-snippets/issues/15

**Comment by dannychapman** [comment]
Agree, but I think we should hit pause on rolling this out until May / June. I dig it though!
